Formula/idevicerestore.rb

```diff
@@ -0,0 +1,34 @@
+class Idevicerestore < Formula
+  desc "Tools to restore and recover iDevices"
+  homepage "https://www.libimobiledevice.org/"
+
+  head do
+    url "https://git.libimobiledevice.org/idevicerestore.git"
```

Comment:
We don't accept head only formula https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Again, nudging @nikias / @FunkyM